conf/backup-with-borg

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+app=$1
```

Comment:
It's really confusing that this var is called `$app` ? I'm guessing we're in fact talking about the name of the borg app / backup method (which may be `borg` or `borg__X` in multi-instance context?)

Can we call this something like `borg_id` or idk